src/lib/components/namespace-select.svelte

```diff
@@ -3,7 +3,7 @@
   import { getContext } from 'svelte';
   import { goto } from '$app/navigation';
 
-  import type { DescribeNamespaceResponse } from '$types/temporal/api/workflowservice/v1/request_response';
+  import type { DescribeNamespaceResponse } from '$types/definitions';
```

Comment:
there is probably a better to do `$types` instead of `$types/definitions`. I didn't focus on this much and wanted to have a working PoC first